### ISSUE TITLE
2019-2020 Preliminary Content Revisions

### ISSUE THREAD

**Issue by rolsanmordua** [UTACM/TexasACM.org#174]
Please update content with following revisions on respective pages. 
- Headings are outlined in square brackets for different parts of pages. 
- Asterisks following by an exclamation mark (*!) are to denote CS 101 blurbs that CS101 leads should revise if they feel the need to.
- Triple asterisks (***) denote changes that will require coordination with other offices and may not be immediately updatable content revisions but are top of mind. 

**About.html > What We Do**

[General Meetings]

Enjoy free food, listen to ACM's announcements, and take a break to mingle with friends and do fun activities. 

[Lecture Series] -> [CS 101] *!
Learn the untaught fundamentals that will help you thrive in intro courses and industry alike. Enjoy a bite while you learn. 

[Family Dinner] -> [Freshmen Social]
Hang out with ACM by grabbing dinner together; make friends and meet upperclassmen! 

[Semesterly Camping Retreat]
Step out of the GDC and UT to go camping!

**About.html > Why Join?**

[Academic]
Our CS101 program is tailored for underclassmen equipping them with the fundamentals they need to succeed inside and outside of school. In addition, our mentorship program matches upperclassmen with freshman, who can help out with the social, academic, and professional aspects of college. Build connections and find your community at UTCS. Learn more [mentorship-sign-up-form-link].

[Network / Professional]

Professional networking opportunities are available to all ACM members. Our General Meetings, Resume Review Office Hours, Resume Database, and Corporate Banquets allow members to meet company recruiters, refine their résumés, and ultimately get interviews and land jobs.

[Leadership Opportunities]
Boost your résumé and learn valuable skills by serving as a leader in the largest CS org on campus. Apply to be an Operational Officer in the Fall and work directly with a Senior Officer on impactful projects while giving back to the community.

**Resources.html > Get Connected**
***Coordinate with Social to update social media reach out means

**Resources.html > CS 101** *!
Missed a workshop or missed a note? View our workshop slides here. 

**Resources.html > Lockers > Cost**
Small: $20
Large: $25

**faq.html > each question (Dynamic Resources backed)**

verify links are not broken

EX: "What sort of events does ACM organize" has broken events link. Same for "How can I get more involved with ACM?" etc.

**Corporate.html > Sponsors**

***Coordinate with Corporate Office to update images of sponsors once list is settled. Remove Citadel and sponsors who did not renew asap.

**Comment by rolsanmordua** [comment]
For CS101 in resources, I think it's worth planning to host content somewhere on the website in some fashion. I am unclear if we want to link to the GitHub repo directly (is it public?) or if we want to setup a dedicated CS101 page that will have content posted as workshops are done (to control premature release of material that hasn't been covered yet which could potentially affect attendance? Not sure if this is a concern at all).

**Comment by rolsanmordua** [comment]
History page should eventually be linked in About I think or even advertised on the carousel once content is ready. Lower priority, though.

**Comment by alex-bellon** [comment]
Waiting on:
- [ ] Get Connected
  - Need to talk with Social Officer
- [ ] CS101 Slides 
  - Need to finish making the slides
- [ ] Sponsors
  - Need to talk with Corporate Officers

**Comment by rolsanmordua** [comment]
I think this one is set to close (?)
